### PR TITLE
added api support for forex channels

### PR DESCRIPTION
**Pull request by shashi165**
Added support for 
1. delete a forex channel
2. create a forex channel

### CODE REVIEW

File: src/admin/api.js [hunk 1]
```diff
@@ -387,6 +387,71 @@ async function createFxpRateForCurrencyChannel(endpoint, currencyPair, rateDetai
...
+            sourceCurrency: sourceCurrency.toUpperCase(),
+            destinationCurrency: destinationCurrency.toUpperCase(),
+        },
+        ...channelDetails,
+    };
+    const currencyChannels = await getFxpCurrencyChannels(endpoint, logger);
+    const targetChannel = currencyChannels
+        .find((currencyChannel) => currencyChannel.sourceCurrency.toLowerCase()
+            === sourceCurrency.toLowerCase()
+        && currencyChannel.destinationCurrency.toLowerCase() === destinationCurrency.toLowerCase());
```
[the remark this answers is not in the document]
Comment (shashi165) [reply src/admin/api.js]: I have copied this from other part of the code, I guess here the problem is the length being restricted to 100 chars. I will investigate

File: src/admin/api.js [hunk 2]
```diff
@@ -387,6 +387,71 @@ async function createFxpRateForCurrencyChannel(endpoint, currencyPair, rateDetai
...
+ * @returns {Promise<*>} The result from the FXP API.
+ */
+async function createFxpCurrencyChannel(endpoint, currencyPair, channelDetails, logger) {
+    const sourceCurrency = extractSourceCurrency(currencyPair);
+    const destinationCurrency = extractDestinationCurrency(currencyPair);
+    const body = {
+        ...{
+            sourceCurrency: sourceCurrency.toUpperCase(),
+            destinationCurrency: destinationCurrency.toUpperCase(),
+        },
```
[the remark this answers is not in the document]
Comment (shashi165) [reply src/admin/api.js]: fixed

File: src/admin/api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (shashi165) [reply src/admin/api.js]: its difficult with the restriction of 100 chars per line, I couldn't format it in a meaningful way

File: package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "ava": "1.0.0-rc.2",
     "chance": "1.0.16",
     "decimal.js": "10.0.1",
-    "eslint": "^6.1.0",
+    "eslint": "^6.8.0",
```
[the remark this answers is not in the document]
Comment (KamuelaFranco) [reply package.json]: I only changed it because it was causing conflict with `react-scripts`, so it's fine in this repo. Also a dev dependency.

**Pull request merged**